src/tscache.ts

```diff
@@ -78,16 +78,16 @@ export class TsCache
 	private syntacticDiagnosticsCache!: ICache<IDiagnostics[]>;
 	private hashOptions = { algorithm: "sha1", ignoreUnknown: false };
 
-	constructor(private noCache: boolean, hashIgnoreUnknown: boolean, private host: tsTypes.LanguageServiceHost, private cacheRoot: string, private options: tsTypes.CompilerOptions, private rollupConfig: any, rootFilenames: string[], private context: RollupContext)
+	constructor(private noCache: boolean, runClean: boolean, hashIgnoreUnknown: boolean, private host: tsTypes.LanguageServiceHost, private cacheRoot: string, private options: tsTypes.CompilerOptions, private rollupConfig: any, rootFilenames: string[], private context: RollupContext)
```

Comment:
`runClean` should also be marked `private`

Reply:
It's not used outside of constructor itself, so no need to make it a member.

Reply:
Oh my bad, I forgot how this syntax worked and thought it was `public` by default (it is not, [docs](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties)). I rarely use classes, sorry about that 😅